README.rst

```diff
@@ -79,5 +79,5 @@ See `License <https://github.com/QCoDeS/Qcodes/tree/master/LICENSE.rst>`__.
     :target: https://travis-ci.org/QCoDeS/Qcodes
 .. |DOCS| image:: https://img.shields.io/badge/read%20-thedocs-ff66b4.svg
    :target: http://qcodes.github.io/Qcodes
-.. |DOI| image:: https://zenodo.org/badge/DOI/10.5281/zenodo.3233612.svg
-   :target: https://doi.org/10.5281/zenodo.3233612
+.. |DOI| image:: https://zenodo.org/badge/DOI/10.5281/zenodo.3266247.svg
+   :target: https://zenodo.org/record/3266247
```

Comment:
I think we should keep this as a link to doi.org. e.g. `https://doi.org/10.5281/zenodo.3266247`

The direct link to zenodo may break but the doi page should always be available

Reply:
thanks! i haven't noticed this!